<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportForm</class>
 <widget class="QWidget" name="ImportForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1" alignment="Qt::AlignRight">
    <widget class="QPushButton" name="pb_exit">
     <property name="text">
      <string>exit</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QListWidget" name="lv_folders"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QProgressBar" name="progb_progress">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="lab_cur_folder">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
